### PR TITLE
Fix compatibility with Proxy-Wasm ABI v0.1.0.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Actually, this should probably include https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/52 when it's merged.

File: source/extensions/common/wasm/context.cc [hunk 1]
```diff
@@ -1136,7 +1141,13 @@ bool Context::validateConfiguration(absl::string_view configuration,
   return result;
 }
 
-absl::string_view Context::getConfiguration() { return configuration_; }
+absl::string_view Context::getConfiguration() {
+  if (plugin_) {
```
Comment (jplevyak) [inline source/extensions/common/wasm/context.cc:1145]: This breaks validateConfiguration.

Comment (PiotrSikora) [reply source/extensions/common/wasm/context.cc:1145]: How come?

Comment (jplevyak) [reply source/extensions/common/wasm/context.cc:1145]: Then let's just remove the configuration_ member.

Comment (PiotrSikora) [reply source/extensions/common/wasm/context.cc:1143]: Done.

**Pull request merged**